package.json

```diff
@@ -5,22 +5,26 @@
   "main": "lib/main.js",
   "scripts": {
     "build": "tsc",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "jest --silent",
+    "test-coverage": "jest --silent --coverage"
   },
   "author": "Shivam Gupta",
   "license": "MIT",
   "devDependencies": {
-    "@types/node": "^13.7.7",
-    "@types/js-yaml": "^3.12.1",
-    "typescript": "^3.8.3"
+    "@types/jest": "25.2.3",
+    "@types/js-yaml": "3.12.1",
+    "@types/node": "13.7.7",
+    "jest": "26.0.1",
+    "ts-jest": "26.0.0",
+    "typescript": "3.8.3"
   },
   "dependencies": {
-    "@actions/core": "^1.2.3",
-    "@actions/tool-cache": "^1.3.1",
-    "download": "^7.1.0",
-    "semver": "^7.1.3",
-    "table": "^5.4.6",
-    "typed-rest-client": "^1.7.2",
-    "js-yaml": "^3.12.1"
+    "@actions/core": "1.2.3",
+    "@actions/tool-cache": "1.3.1",
+    "download": "7.1.0",
+    "semver": "7.1.3",
+    "table": "5.4.6",
+    "typed-rest-client": "1.7.2",
+    "js-yaml": "3.12.1"
```

Comment:
Are all these versions the latest?

Reply:
No, I only removed the greater that signs from the versions. Should I replace all these with the latest versions?